cmd/loop/loopout.go

```diff
@@ -41,6 +42,13 @@ var loopOutCommand = cli.Command{
 			Name:  "amt",
 			Usage: "the amount in satoshis to loop out",
 		},
+		cli.Uint64Flag{
+			Name: "htlc_confs",
```

Comment:
nit: I wonder if naming this `htlc_conf_target` would be better? Just so that it aligns with the `conf_target` flag. Maybe it's even worse as then users might mix the two.

Reply:
These two have slightly different meaning. The `conf_target` flag is the number of confirmations we target in fee estimations for our sweep vs the `htlc_confs` which is the number of blocks that we definitely will wait for the htlc to confirm on chain.

Perhaps `conf_target` should actually be `sweep_conf_target` but I'm not sure it's worth potentially breaking some scripts that people may have out there by changing  the name.

Reply:
Yeah, maybe `sweep_conf_target` might be a better name for the `conf_target` but also wouldn't change it.